src/Styles/Profile.styles.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components"
+
+export const UserInfoContainer = styled.div<{ mb?: number }>`
+  padding: 0.8rem 1.2rem;
+  font-size: 1.6rem;
+  font-weight: 500;
+  background-color: var(--gray-20, #f2f4f6); //TODO: 컬러 변수로 등록하기
+  border-radius: 8px;
+  border: 1px solid var(--gray-20, #f2f4f6); //TODO: 컬러 변수로 등록하기
+  margin-bottom: 0.5rem;
+  color: #4e5968; //TODO: 컬러 변수로 등록하기
```

Comment:
요부분도 다른 변수 설정과 같은 코멘트입니다!👍
#4e5968 색상은 theme.ts에 없어서 일단 white로 제안드렸습니다!
```suggestion
  color: ${({ theme }) => theme.colors.white}; 
```